tests/develop/error_collection-003.phpt

```diff
@@ -8,7 +8,7 @@ xdebug.file_link_format=xdebug://%f@%l
 xdebug.filename_format=
 xdebug.collect_assignments=1
 xdebug.collect_return=1
-xdebug.var_display_max_data=2048
+xdebug.var_display_max_data=3072
```

Comment:
Hah - did the error messages get so much bigger? I wonder whether this should just be `-1` instead, in case they get even bigger?

Reply:
Could be related to long path used in my build system, and SCL packages (e.g. /opt/remi/php80/root/usr/bin/php)
BTW, only a few bytes are missing, and I increase value a lot ;)

Reply:
changed to -1